composer.json

```diff
@@ -30,6 +30,8 @@
         "lukasoppermann/http-status": "^2.0",
         "psr/http-factory": "^1.0",
         "psr/http-server-middleware": "^1.0",
+        "slim/psr7": "^1.1",
+        "slim/slim": "4.*",
```

Comment:
Because we only need slim only for the example I think its better to remove it from here or move this to the dev requirements.